### DISCUSSION TITLE
子应用的弹窗如何覆盖基座应用

### DISCUSSION THREAD

**Discussion by hellocy23** [jd-opensource/micro-app#292] [Q&A]
![微信截图_20220224165654](https://user-images.githubusercontent.com/12553354/155491737-a110fa2f-bc9f-4126-b8bc-0eb0d97ee52f.png)

**Comment by bailicangdu** [comment]
检查一下是不是postion定位的问题

**Reply by hellocy23** [reply]
解决了，是我子应用外层div z-index的层级问题
